libminifi/src/capi/api.cpp

```diff
@@ -32,6 +32,7 @@
 using string_map = std::map<std::string, std::string>;
 
 class API_INITIALIZER {
+ public:
```

Comment:
Why was this needed?

Reply:
Not needed, just left there, will remove, good point.